docs-2.0/3.ngql-guide/10.tag-statements/1.create-tag.md

```diff
@@ -0,0 +1,84 @@
+# CREATE TAG
+
+```ngql
+CREATE TAG [IF NOT EXISTS] <tag_name>
+    ([<create_definition>, ...])
+    [tag_options]
+
+<create_definition> ::=
+    <prop_name> <data_type> [NULL | NOT NULL]
+
+<tag_options> ::=
+    <option> [, <option> ...]
+
+<option> ::=
+    TTL_DURATION [=] <ttl_duration>
+    | TTL_COL [=] <prop_name>
+    | DEFAULT <default_value>
+```
+
+`CREATE TAG` creates a tag with the given name in a graph space. You must have the `CREATE` privilege for the graph space. (TODO: privilege doc) To create a tag in a specific graph space, you must use the graph space first.
+
+## Tag name
+
+- `IF NOT EXISTS`: Creating an existent tag results in an error. You can use the `IF NOT EXISTS` option to conditionally create the tag and avoid the error.
+
+    > **NOTE**: The tag existence detection here compares only the tag names (excluding properties).
+- `tag_name`: The tag name must be **unique** in a graph space. Once the tag name is set, it can not be altered. The rules for permitted tag names are the same as those for graph space names. See Schema Object Name (TODO) for detail.
+
+### Property names and data types
+
+- `prop_name`
+
+    `prop_name` is the name of property. It must be unique for each tag.
+
+- `data_type`
+
+    `data_type` shows the data type of each property. For a full description of the property data types, see [Data types](../3.data-types/1.numeric.md).
+
+- `NULL | NOT NULL`
+
+    Specifies if the property supports `NULL | NOT NULL`. The default value is `NULL`.
+
+- `DEFAULT`
+
+    Specifies a default value for a property. The default value can be any of the data type or expressions supported by Nebula Graph. If no value is specified, the default value is used when inserting a new vertex.
```

Comment:
> The default value can be any of the data type or expressions supported by Nebula Graph.

A value cannot be a data type. (And type should be in the plural form.) And please consider reorganizing this sentence.